jellyfish/room_api.py

```diff
@@ -0,0 +1,73 @@
+"""
+    RoomApi used to manage rooms
+"""
+
+import openapi_client as jellyfish_api
+
+from openapi_client import (AddPeerRequest, AddComponentRequest, PeerOptions,
+                            ComponentOptions, Room, RoomConfig, Peer, Component)
+
+
+class RoomApi:
+    """Allows for managing rooms"""
+
+    def __init__(self, server_address: str, server_api_token: str):
+        self._configuration = jellyfish_api.Configuration(
+            host=server_address,
+            access_token=server_api_token
+        )
+
+        self._api_client = jellyfish_api.ApiClient(self._configuration)
+        self._room_api = jellyfish_api.RoomApi(self._api_client)
+
+    def create_room(self, max_peers: int = None, video_codec: str = None) -> (str, Room):
+        """Creates a room"""
```

Comment:
I don't know where we're planning to put the documentation (when everything's finished), but I think there should be some explanation of the return values in the docstring here

Now that I look at it, there should really be something like that in our elixir sdk as well...

Reply:
Yeah I thought about it as well. I decided to leave it for later though. We have to find a tool for generating docs from docstrings. We may also decide to use some additional typing libraries.